docs/release-process.md

```diff
@@ -257,7 +257,7 @@ slightly different steps. We'll call attention where the steps differ.
    For example, if the RC is `8.2.1RC1` then the version numbers in the version
    branch should be bumped to `8.2.2-dev`. We do this regardless of whether we
    build a new RC to make sure `version_compare()` works correctly. See
-   [Bump for 8.1.8-dev][] for a real example.
+   [Bump for 8.3.21-dev][] for a real example.
```

Comment:
The commit title (i.e. "PHP 8.3 is now for PHP 8.3.21-dev") does not match this title (i.e. "Bump for 8.3.21-dev"). So it could still be confusing

Reply:
Good point. Fixed.